api/android/README.md

```diff
@@ -125,6 +125,7 @@ You can download these files from [nnsuite testcases repository](https://github.
 ```
 # You must put the below model and label files in the internal storage of your Android target device.
 {INTERNAL_STORAGE}/nnstreamer/test/mobilenet_v1_1.0_224_quant.tflite
+{INTERNAL_STORAGE}/nnstreamer/test/add.tflite
```

Comment:
This will be added to https://github.com/nnsuite/testcases/tree/master/DeepLearningModels/tensorflow-lite/Mobilenet_v1_1.0_224_quant

Reply:
https://github.com/nnsuite/testcases/pull/14

Reply:
Above PR merged